### PR TITLE
Wc 166/configure postcss

### PR DESCRIPTION
**Pull request by akdetrick**
#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-166

#### Description
- adds webpack configuration for PostCSS and CSS modules
- moves styling for `bounds` into a css module as example
- fix issues with custom properties & color functions
- fix issues with custom properties & sass calculation
- removes `customPropertyValue` usage
- removes `--responsiveSpace` custom property usage in favor of `--responsive-space` from constants

### CODE REVIEW

File: assets/scss/components/_forms.scss [hunk 1]
```diff
@@ -30,6 +32,21 @@ input:focus + .icon--field {
 	font-weight: normal;
 }
 
+.label--required:after {
+	content: attr(data-requiredtext);
+	color: $C_textSecondary;
+	// font-size: $font-size-tiny;
```
Comment (mperrotti) [inline assets/scss/components/_forms.scss]: probably not part of this PR, but let's remove this

File: assets/scss/components/_forms.scss [hunk 2]
```diff
@@ -30,6 +32,21 @@ input:focus + .icon--field {
 	font-weight: normal;
 }
 
+.label--required:after {
```
Comment (mperrotti) [inline assets/scss/components/_forms.scss]: Why aren't we replacing SASS vars with custom properties in here?

Comment (akdetrick) [reply assets/scss/components/_forms.scss]: rather than replacing all Sass vars, I remapped them in sasstools v4:
https://github.com/meetup/swarm-sasstools/compare/master...release/4.0.0

All existing Sass vars now map to custom properties, which allows us to launch faster and clean up in steps afterward.

Comment (mperrotti) [reply assets/scss/components/_forms.scss]: I completely forgot about that 🤦‍♂️

**Comment by akdetrick** [comment]
Adding "prelim" until we investigate a plan to launch to `mup-web` without major breaking changes. We may need to undo some things in this branch before merging.

**Comment by akdetrick** [comment]
Not merging to master; this should happen in a non-breaking way.
